magento/pwa-studio
# [docs] content rendering topic

## Description

This PR creates a new topic covering content rendering and how it relates to PWA Studio and UPWARD.

## Related Issue

Closes PWA-160

## Acceptance 
@zetlen 
@ericerway 
@awilcoxa 
### Verification Stakeholders
@ericerway 
@awilcoxa 
### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
2. Lint the file: `npm run lint src/technologies/basic-concepts/content-rendering`
3. Verify no linting errors
4. Build the HTML preview: `npm run develop`
5. Navigate to `<url>/technologies/basic-concepts/content-rendering`
6. Verify page and content exists
7. Verify table of contents is listed on the left nav and highlighted

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
